Scemantic Focus

```diff
@@ -0,0 +1,110 @@
+---
+id: 
+name: Scemantic Focus 
+rule_type: atomic
+
+description: | 
+ This rule checks that every element part of the sequential focus navigation has a semantic role.
+
+accessibility_requirements: 
+  wcag20:2.4.4: # Link Purpose (In Context) 
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+- DOM Tree
+
+authors:
+- Carlos Duarte
+- Damien Engels
+- Marie Trudelle 
+- Ramit Garg
+---
+
+## Applicability
+
+The rule applies to all element in the sequential focus navigation excepts scrollable content element.
```

Comment:
- "all elements" is pretty vague. I propose "all HTML and SVG elements"
- "sequential focus navigation" must be defined
- "scrollable content element" must be defined

Reply:
`[sequential focus navigation](https://www.w3.org/TR/html/editing.html#sec-sequential-focus-navigation`